### ISSUE TITLE
FYI - Microsoft Office UI Fabric components, accessibility announcer

### ISSUE THREAD

**Issue by danielweck** [readium/r2-testapp-js#20]
Good alternative to Material UI design? (looks pretty polished, with good screen reader support)

https://developer.microsoft.com/en-us/fabric/#/components/announced

https://github.com/OfficeDev/office-ui-fabric-react

... as used in the "accessibility-insights-web" `axe-core` -based accessibility checker (Chrome extension):

https://github.com/Microsoft/accessibility-insights-web/blob/055cfac2f419a47aac0cf85044cd7a5fcb93e57a/package.json#L92

**Comment by danielweck** [comment]
Uncertainty about Chromium support (for both Electron and future-Edge):

"We focus on Edge + Narrator compatibility which seems to be working as expected in the above scenarios (see output below).

Since it works in Narrator, we are likely providing the correct attributes and this is more likely an issue with NVDA / JAWS integration and should be raised in their bug report channels"

https://github.com/OfficeDev/office-ui-fabric-react/issues/8294

**Comment by danielweck** [comment]
See also: https://github.com/readium/r2-navigator-js/issues/37
